packages/dashboard/src/portal/components/VotePopup.vue

```diff
@@ -20,22 +20,33 @@
 <script lang="ts">
 import { Component, Vue } from "vue-property-decorator";
 
+import { IFarm, INode } from "../../explorer/graphql/api";
+import config from "../config";
+import { getFarm } from "../lib/farms";
+
 @Component({
   name: "VotePopup",
 })
 export default class VotePopup extends Vue {
   openProposalDialog = false;
   proposals = 0;
   voteMsg = "You have a pending proposal to vote!";
-
-  private __destroied = false;
+  $api: any;
```

Comment:
the api type here is `ApiPromise` you can import it

`import { ApiPromise } from "@polkadot/api"`